app/controllers/nsm/steps/youth_court_claim_additional_fee_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Nsm
+  module Steps
+    # :nocov:
+    class YouthCourtClaimAdditionalFeeController < Nsm::Steps::BaseController
+      def edit
+        @form_object = YouthCourtClaimAdditionalFeeForm.build(
+          current_application
+        )
+      end
+
+      def update
+        # update_and_advance(YouthCourtClaimAdditionalFeeForm, as: :youth_court_claim_additional_fee)
+      end
+
+      private
+
+      def decision_tree_class
+        Decisions::DecisionTree
+      end
+
+      def additional_permitted_params
+        [:youth_court_fee_claimed]
```

Comment:
I'm surprised you need this - it should be inferred from being part of the `YouthCouldClaimAdditionalFeeForm.attributes`